### PR TITLE
docstrings for merge.py

### PR DESCRIPTION
**Pull request by skaae**

### CODE REVIEW

File: lasagne/layers/merge.py [hunk 1]
```diff
@@ -11,6 +11,24 @@
 
 
 class ConcatLayer(MultipleInputsLayer):
+    """
+    Concatenates multiple inputs along the specified axis. Inputs should have
+    the same shape except for the dimension specified in axis, which can have
+    different size.
```
Comment (benanne) [inline lasagne/layers/merge.py]: different sizes*

**Comment by benanne** [comment]
looks good to me, just spelling :)

**Comment by skaae** [comment]
fixed spelling error.

**Comment by benanne** [comment]
... and merging!

**Pull request merged**